main.cpp

```diff
@@ -78,44 +82,92 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
-
+ */
+int calculateVolume(int lenght =30, int wide = 20)
+{
+        ignoreUnused(lenght, wide); 
+        int vol = 0;
+        vol = lenght * wide * wide;
+        return{vol};
+}
 /*
  2)
  */
+bool chooseInstrument(int playTime , int instrumentType = 0 )
+{
+            ignoreUnused(playTime, instrumentType);
+            return{};
+}
 
 /*
  3)
  */
-
+float getTempo(float rhythym = 3.2f, int bars = 4)
+{
+    ignoreUnused(rhythym, bars);
+    return{rhythym * bars};
+}
 /*
  4)
  */
+double calcTemperature(int type = 0, double degrees =36.0)
+{
+    ignoreUnused(type, degrees);
+    return{};
+}
 
 /*
  5)
  */
-
+bool solutionDone(double result = 0, int type = 1)
+{
+    ignoreUnused(result,type);
+    return{};
+}
 /*
  6)
  */
-
+double getAmplification(double initialValue=1, double finalValue=100)
+{
+    ignoreUnused(initialValue, finalValue);
+    return{finalValue/initialValue};
+}
 /*
  7)
  */
+int getNumberCables(int sources, int outputs= 30)
+{
+    ignoreUnused(sources, outputs);
+    int cables = 0;
+    cables = (outputs - sources)*2;
+    return {cables};
+}
```

Comment:
`cables` is an `int`.  you don't need to wrap it in `{ }` when returning it. 
You're supposed to just write `return { };`
- [ ] revise